test.jl

```diff
@@ -0,0 +1,23 @@
+using RecursiveArrayTools, BenchmarkTools
+
+function Base.copyto!(A::ArrayPartition,src::ArrayPartition)
```

Comment:
what's this file for?

Reply:
sorry just used for benchmarking, will remove.